include/myst/tcall.h

```diff
@@ -18,8 +18,6 @@
 typedef enum myst_tcall_number
 {
     MYST_TCALL_RANDOM = 2048,
-    MYST_TCALL_ALLOCATE = 2049,
-    MYST_TCALL_DEALLOCATE = 2050,
```

Comment:
Should we shift the following syscall numbers upward so there is no "holes" in them?

Reply:
Done. Thanks.